data/themes/darktable.css

```diff
@@ -866,12 +889,15 @@ menuitem arrow,
 tooltip,
 popover,
 combobox window,
-dialog combobox window
+dialog combobox window,
+popover #bauhaus-popup,
```

Comment:
Shouldn't this be only #bauhaus-popup ? And popover removed ? Otherwise all the standard combos in modules still have a black background, see for example in exposure module:

![image](https://user-images.githubusercontent.com/467069/99871797-9905e800-2bdd-11eb-86a1-664702a25601.png)

And note that the #bauhaus-popup also control the right-click menu over the sliders. So we can really makes all popup looks alike.

Two other things:

The menu on the module groups seems to have a double border, one around the window enclosing the scrollbar and on inner border around the content:

![image](https://user-images.githubusercontent.com/467069/99871857-fb5ee880-2bdd-11eb-9991-2a61dc6b9d0e.png)

And the menu on bottom of darkrrom, lighttable seems to have a black border on top and bottom:

![image](https://user-images.githubusercontent.com/467069/99871889-1cbfd480-2bde-11eb-968d-c53ac16a4542.png)

Reply:
As I said, it's not possible to apply a background color on right-click popover in sliders. I've tried and if I remove popover on that line, the only thing I add is that background-color of sliders change, but not right-click popover background. And that makes just modules awful. So popover is really needed. The only way to add background-color would be to update Gtk code to allow that and have a specific tag for only the right-click popover menu on sliders (and avoid to update sliders background the same way. So not CSS here.

About black border, I have seen that but I don't know where they are. You could see that I add that to same settings and borders than all other menus. And those others menus have a thin clearer border all around. And that is not applied there even if it's set. I agree, some improvement needed but not with CSS here. It's some piece of Gtk code to update to allow border setting to be applied here.

Last thing remaining and yes probably CSS, it's the double border. Will check that.

Reply:
Forget what I've said just above about right-click in sliders. I made a new test and don't understand what I've done before. It works indeed great without the popover menu. Anyway, applying borders in bauhaus widget is impossible. I see them applied with Gtk inspector tool, so CSS is ok but they are not applied. Something in Gtk code.

Those bauhaus have been improved but they remains awful to work on.